draft-irtf-cfrg-vdaf.md

```diff
@@ -1262,6 +1283,12 @@ public shares, input shares, prep shares, and prep messages. Minimally:
   Vdaf.PrepMessage` decodes a prep message, using the prep state as optional
   decoding context.
 
+* `Vdaf.decode_agg_param(encoded: bytes) -> Vdaf.AggParam` decodes an
+  aggregation parameter.
+
+* `Vdaf.encode_agg_param(agg_param: Vdaf.AggParam) -> bytes` encodes an
```

Comment:
We should adjust the definition of `encode_agg_param` for Poplar1 in 8.2.6.6 to match this, so it takes in the aggregation parameter as a single tuple argument (rather than two separate arguments).